assets/js/src/newsletters/types/automatic_emails/event.jsx

```diff
@@ -33,6 +33,7 @@ class AutomaticEmailEvent extends React.PureComponent {
           className="button button-primary"
           disabled={disabled}
           onClick={!disabled ? _.partial(this.props.eventsConfigurator, event.slug) : null}
+          data-automation-id={event.slug}
```

Comment:
Maybe we could add some prefix here e.g. `setup-button-` What do you think?

Reply:
I think prefixing it with `create_` would be more consistent with what we have on newsletter types page (`create_standard`, `create_woocommerce`, ...).

Reply:
I agree some prefix would be good. Those ids should be unique so the longer the better.